sanicbot/core/cogs.py

```diff
@@ -0,0 +1,47 @@
+import httpx
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from sanicbot.core.utils import failure_message, success_message
+
+
+class GitCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.org_url = 'https://github.com/sanic-org/'
```

Comment:
What is this used for? Looks like we include the same information down below in `lookup`